### PR TITLE
Make gallery thumbnail clickable, gallery can now be run locally

### PR DESCRIPTION
**Pull request by Tehsurfer**
Made a few changes:

 1. Can now run this app stand alone ( see package.json for new build settings)
 2. Reduce the number of required inputs (more defaults)
 3. Gallery thumbnails now clickable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,18 @@
 {
-  "name": "@abi-software/gallery",
-  "version": "0.3.1",
+  "name": "@/abi-software/gallery",
```
Comment (hsorby) [inline package.json]: I don't think that this is the correct way of naming the package, I think the original name is correct.

Comment (alan-wu) [reply package.json]: @Tehsurfer Can you please correct this please?

File: src/components/Card.vue [hunk 2]
```diff
@@ -46,11 +46,13 @@
 <script>
 // import { SvgIcon } from '@abi-software/svg-sprite'
 import Vue from 'vue'
-import { Button, Card, Popover } from 'element-ui'
+import { Button, Card, Popover, Tooltip, Loading } from 'element-ui'
```
Comment (alan-wu) [inline src/components/Card.vue:49]: Is tooltip used anywhere?

Comment (alan-wu) [reply src/components/Card.vue:49]: Same for loading.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Need to include few minor changes.

File: src/components/Card.vue (hunk 2 above)
Comment (Tehsurfer) [reply src/components/Card.vue:49]: @alan-wu Yeah, it is used right here:
https://github.com/ABI-Software/mapcore-gallery/blob/2eab349f9918ad6aa43f37004aba178100dbd3a7/src/components/Card.vue#L14

File: package.json (hunk 1 above)
Comment (Tehsurfer) [reply package.json]: Should be fixed now

**Review by alan-wu — APPROVED** [review]
Looks good.

**Pull request merged**